src/clj/server/csv.clj

```diff
@@ -0,0 +1,37 @@
+(ns server.csv
+  (:require
+   [clojure.string :as string]
+   [shoreleave.middleware.rpc :refer [defremote]]))
+
+(defremote csv-files
```

Comment:
I'm don't know what you mean. The existing code is already in a clojure file.

I don't understand why "write-csv-row!", "write-csv-rows!", and "start-csv-file" are remote callbacks. I think these are too low level to get exposed as callbacks. 

I'm at fault for not knowing how the defremotes work, but the only way to interact with them is via a callback, yes? This would make testing these methods harder than if they were functions.